caffe2/perfkernels/CMakeLists.txt

```diff
@@ -89,3 +89,14 @@ set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS} PARENT_SCOPE)
 set(Caffe2_DEPENDENCY_WHOLE_LINK_LIBS
     ${Caffe2_DEPENDENCY_WHOLE_LINK_LIBS}
     PARENT_SCOPE)
+
+# For iOS x86_64 build, the libCaffe2_perfkernels_avx2.a is not installed properly.
+# In the future, this file might be removed from mobile.  This is just a temporary fix 
+# until we find a better solution.
```

Comment:
this feels weird. Mostly like we don't want to run or build avx2 instructions, even in simulator. Perfkernels have a fallback mechanism for plain implementation, probably we should just sprinkle some flags there to "detect" that compiler doesn't support AVX if it's a simulator. See docs in https://github.com/pytorch/pytorch/blob/master/caffe2/perfkernels/common.h . Most likely, we can disable AVX2 build here: https://github.com/pytorch/pytorch/blob/8cd6d2f1015fa4e7a63d62471c7202cea77dbc27/cmake/MiscCheck.cmake#L163-L173

Reply:
@dzhulgakov I've fixed this, somehow when I run ghstack, it generates a new PR.So I'll close this one, will tag you in the new one.